components/automate-gateway/api/nodes/manager/manager.proto

```diff
@@ -122,6 +199,23 @@ service NodeManagerService {
 			action: "infra:nodeManagers:list"
 		};
 	};
+
+	/*
+	Search node fields
+
+	Searches the available values for a given field and nodemanager id.
```

Comment:
Not sure "values" is a useful word there; might "nodes" be more appropriate (to mirror what "Search Nodes" uses) ?